backend/wmg/data/snapshot.py

```diff
@@ -83,7 +83,11 @@ def _load_snapshot(new_snapshot_identifier) -> WmgSnapshot:
 
 
 def _open_cube(cube_uri) -> Array:
-    return tiledb.open(cube_uri, ctx=create_ctx(json.loads(WmgConfig().tiledb_config_overrides)))
+    config = WmgConfig()
+    tiledb_config_overrides = config.tiledb_config_overrides
+    if type(tiledb_config_overrides) is str:
```

Comment:
Why do we need a type check here? Is it an issue with the default template? I think it would make more sense to change it there for consistency

Reply:
I tried changing the empty dict in the defaults template to "{}" but it broke the API elsewhere. I'll investigate further.